mobile/src/components/course/components/course-modules/course-modules.tsx

```diff
@@ -12,10 +21,18 @@ type Props = {
 };
 
 const CourseModules: FC<Props> = ({ courseModules, isLoading }) => {
+  const navigation = useAppNavigate();
+  const dispatch = useAppDispatch();
+
   if (!courseModules.length) {
     return <></>;
   }
 
+  const handleModulePress = (courseId: number, moduleId: number): void => {
+    dispatch(courseModulesActions.getModuleById({ courseId, moduleId }));
+    navigation.navigate(AppScreenName.COURSE_MODULE);
+  };
```

Comment:
pass this via props

Reply:
I meant you shouldn't have dispatch or navigation inside course-modules.tsx and get handler from props

Reply:
Oh...
I understood